core/src/main/resources/defaults/blueprints/7.2.9/cdp-opdb.bp

```diff
@@ -173,12 +173,12 @@
             "base": true
           },
           {
-              "refName": "hbase-HBASETHRIFTSERVER-BASE",
-              "roleType": "HBASETHRIFTSERVER",
-              "displayName": null,
-              "configs": [],
-              "base": true
-           },
+            "refName": "hbase-HBASETHRIFTSERVER-BASE",
```

Comment:
why is this changed?

Reply:
That's just spacing.

Reply:
please revert back that part because this is not related to the change

Reply:
done